From 6102e8968641c92988b5e9db17f004c19c5b9484 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 19 Sep 2011 12:20:55 +0200
Subject: [PATCH] 99base/init: remove /dev/root helper symlink

Any tool relying on /dev/root has to be fixed.
---
 modules.d/99base/init |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 30a5cf9..a759c44 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -447,6 +447,9 @@ fi
 
 wait_for_loginit
 
+# remove helper symlink
+[ -h /dev/root ] && rm -f /dev/root
+
 info "Switching root"
 
 unset PS4
